### PR TITLE
docs: update changelog and whatsnew

### PR DESCRIPTION
**Pull request by MuyangAmigo**
ms-teams-vscode-extension	v4.1.0-rc.1

CLI BREAKING CHANGE:
  

Extension-toolkit BREAKING CHANGE:
 

CLI feat commits:
  

Extension-toolkit feat commits:

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 4.1.0 - Oct 17, 2022
+
+Incremental version for Teams Toolkit with new features and several enhancements.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:5]: This line can be removed.

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 4.1.0 - Oct 17, 2022
+
+New features:
+
+- New scenario-based Teams app template `Workflow bot` that enables you to create workflow style Teams application to facilitate users progress through multiple Adaptive Cards via buttons and inputs. Checkout our [documentation](https://aka.ms/teamsfx-card-action-response) for more details.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:7]: Use the new Workflow bot template to create sequential workflows where Adaptive Cards can be returned in response to user input. Learn more about this new template in the [documentation](https://aka.ms/teamsfx-card-action-response).

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+- Debugging process is now transparently defined in `.vscode/tasks.json` and accepts parameters and arguments to enable customization and provide more granular control over debugging processes. Checkout our [documentation](https://aka.ms/teamsfx-debug-tasks) for more details. If you are running an existing project created by older Teams Toolkit versions, checkout this guide for [how to upgrade](https://aka.ms/teamsfx-debug-upgrade-new-tasks).
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:8]: Configure each step of the debugging experience in Teams Toolkit by customizing the `.vscode/tasks.json` file, which now [surfaces parameters for finer-tuned control](https://aka.ms/teamsfx-debug-tasks).

File: packages/vscode-extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+- Simplified single sign-on for Message Extension and Command bot via `Add sso` command and TeamsFx SDK that helps you leverage logged-in Teams user token when accessing Microsoft Graph from your Teams application. Checkout our [documentation](https://aka.ms/teamsfx-add-sso) for more details.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:9]: Use the new **Add SSO** command for a simple way to [configure single sign-on for Message extension and Command bot projects](https://aka.ms/teamsfx-add-sso).

File: packages/vscode-extension/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
...
+- SharePoint developers can now easily combine relevant web parts in a single personal Teams app as a [multi-tab Teams app](https://learn.microsoft.com/sharepoint/dev/spfx/build-for-teams-me-experience#build-a-multi-tab-personal-teams-app) via `Add feature`.
```
Comment (therealjohn) [inline packages/vscode-extension/CHANGELOG.md:10]: SPFx developers can now combine web parts as a [multi-Tab Teams app](https://learn.microsoft.com/sharepoint/dev/spfx/build-for-teams-me-experience#build-a-multi-tab-personal-teams-app) using the **Add Feature** menu.

File: packages/vscode-extension/WHATISNEW.md [hunk 6]
```diff
@@ -22,6 +23,20 @@
 
 # Changelog
 
+## 4.1.0 - Oct 17, 2022
```
Comment (therealjohn) [inline packages/vscode-extension/WHATISNEW.md:26]: Same edits as above.

**Review by therealjohn — CHANGES_REQUESTED** [review]
What's the purpose of having a CHANGELOG and WHATISNEW with the same content?

**Comment by MuyangAmigo** [comment]
> What's the purpose of having a CHANGELOG and WHATISNEW with the same content?

@zyxiaoyuer Yu do you have context of this?

**Pull request merged**